### ISSUE TITLE
Create neuronal network data Pali

### ISSUE THREAD

**Issue by ayya-vimala** [ayya-vimala/pali-networks#8]
* Run the neuronal network for pali data only.
* Create csv files
* check csv files and feedback

**Comment by sebastian-nehrdich** [comment]
I just uploaded a fresh batch of Pali data under suttas/pali. I noticed that about 180 files are missing in the output, I didn't yet find out why this is happening...

**Comment by ayya-vimala** [comment]
Wow, that's great. Will have a look soon. Yes, please have a look at what is wrong with those 180 files ... I will have a look also and see if I can find out anything.

**Comment by sebastian-nehrdich** [comment]
I fixed it, the dataset now is pretty much complete. I think *maybe* one or two files got lost, probably because they ran into a loop while finding quotations and thus stalled. Anyway, not too dramatic I think...

**Comment by ayya-vimala** [comment]
I had a look. Looks very good as far as I can see, but a bit difficult at the moment now everything seems to be not working anywhere. I adapted a branch to see a bit of it but it would require me to do a lot of bugfixing so I think I will leave that until we can do this via the API.

**Comment by sebastian-nehrdich** [comment]
Yes the only branch that could display the data at the moment might be 102. But I guess we have to do some serious code refactoring, especially because we need to 'bring back' the formatted display of the different languages. 102 is all messy at the moment (and this is to a good degree my fault). I agree that it's possibly best to put our energy into the api migration and work from there, otherwise we end up doing all the work twice with not much benefit...

**Comment by ayya-vimala** [comment]
Just one little thing I noticed: language = "pli" (instead of "pali") Not a major thing but this is how we have it elsewhere to be consistent with the 3-letter codes of other languages. No need to change that now but maye for the next run.

**Comment by sebastian-nehrdich** [comment]
Yes I agree, we discussed it before and I was not attentive here. having three-letter codes everywhere is indeed better. I think with these jsons it should be fine to just run over tem with a perl-script and replace the wrong lang-tags, I will do that tomorrow & reupload the jsons!

**Comment by TrebuhD** [comment]
what's the status on this issue?
